merge-and-build.sh

```diff
@@ -76,11 +76,11 @@ git merge master -m "Merge master into enterprise-${ose_version}"
 cd "${workpath}"
 git clone git@github.com:openshift/ose.git
 cd ose
-git add remote upstream git@github.com:openshift/origin.git
-git fetch --all
+git remote add upstream git@github.com:openshift/origin.git
+git fetch upstream master
 git merge -m "Merge remote-tracking branch upstream/master" upstream/master
 # Pull in the origin-web-console stuff
-vc_commit="$(GIT_REF=master hack/vendor-console.sh 2>/dev/null | grep "Vendoring origin-web-console" | awk '{print $4}')"
+vc_commit=$(GIT_REF=master hack/vendor-console.sh | awk '/Vendoring origin-web-console/{print $4}')
```

Comment:
`2>/dev/null` might make debugging errors in `hack/vendor-console.sh` difficult (and it doesn't influence the output capture for `vc_commit`).  Also simplifying grep/awk to just awk.